server/utils/dateFormats.ts

```diff
@@ -0,0 +1,8 @@
+import formatISO from 'date-fns/formatISO'
+
+// eslint-disable-next-line
+export class DateFormats {
```

Comment:
Good call on this! 👍

Reply:
Thanks, I'll add the other date formats in a seperate PR tomorrow!